Dockerfile

```diff
@@ -1,10 +1,10 @@
-FROM silintl/php8:8.1
-LABEL maintainer="matt_henderson@sil.org"
+FROM silintl/php8:8.3
```

Comment:
Check for the model generation scope and data typing issues with Yii 2.0.52.

Reply:
@mtompset is that a breaking change? What happens if no change is made?